tox.ini

```diff
@@ -0,0 +1,74 @@
+[tox]
+minversion = 2.9.1
+basepython = python3
+skip_missing_interpreters = True
+#skipsdist=True  # Not sure this is needed yet
+envlist =
+    pep8
+    pylint_minimal
+    pytest
+# Disabled as these tests are already covered above and Leap 15.2 tox runs
+# py3{8,9} as py36
+#    py35
+#    py36
+#    py37
+#    py38
+#    py39
+
+[testenv]
+description = Python
+usedevelop = True
+deps =
+    -rrequirements-test.txt
+commands =
+    pytest {posargs}
+
+[testenv:pep8]
+description = pep8
+usedevelop = True
+envdir = {toxworkdir}/test
+deps =
+    -rrequirements-test.txt
+commands =
+    flake8 imagenameparser 
+
+[testenv:pylint]
+description = Linter (disable snake_case check)
+usedevelop = True
+envdir = {toxworkdir}/test
+deps =
+    -rrequirements-test.txt
+commands =
+    # disable snake_case
+    pylint --disable=C0103 --max-module-lines=1100 imagenameparser 
+
+[testenv:pylint_minimal]
+description = Linter (disable snake_case check)
+usedevelop = True
+envdir = {toxworkdir}/test
+deps =
+    -rrequirements-test.txt
+commands =
+    # disable snake_case
+    pylint --disable=C0103 --max-module-lines=1100 imagenameparser 
+[testenv:pytest]
```

Comment:
should we add coverage ? if we can leverage the pint name check, we should always hit 100% of every image name, otherwise we should not pass the CI. This will prevent future "corner" cases if anything changes in the image name scheme standard.

Reply:
I suggest that that can be done as a follow on activity after getting the basic implementation landed.